coralproject/talk-fe
# Add default style

## What does this PR do?

Adds a default style to the embeddable comment stream.

## How do I test this PR?

1. Install talk-fe according to instructions in the readme
2. `cd packages/coral-embed-stream`
3. `npm run dev-start`
4. Embeddable stream should have basic styling.

## Notes

- This styling is a placeholder pending input from the design team.
